Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4014,10 +4299,10 @@ def to_context(self):
                 domain_context['Tags'] = self.tags
 
             if self.feed_related_indicators:
-                feed_related_indicators = []
-                for feed_related_indicator in self.feed_related_indicators:
-                    feed_related_indicators.append(feed_related_indicator.to_context())
-                domain_context['FeedRelatedIndicators'] = feed_related_indicators
+                domain_context['FeedRelatedIndicators'] = self.create_context_table(self.feed_related_indicators)
+
+            if self.whois_records:
+                domain_context['WhoisRecords'] = self.create_context_table(self.whois_records)
```

Comment:
Why is this the only change in the 'to_context' function of the Domain class? 
I mean why aren't new other fields such as 'description', 'stix_id', 'blocked' added here?

Reply:
Done